src/language-server/project/client.ts

```diff
@@ -412,7 +412,7 @@ export class GraphQLClientProject extends GraphQLProject {
                     this.frontendUrlRoot ?? "https://studio.apollographql.com";
 
                   const variant = this.config.variant;
-                  const graphId = this.config.graph;
+                  const graphId = this.studioGraphId;
```

Comment:
If I have a config file like
```
module.exports = {
  client: {
    service: {
      name: "localSchema",
      localSchemaFile: "./starwarsSchema.graphql",
    },
  },
};
```
`graphId` would be `localSchema` - of course, linking to studio for that is kinda useless, but that's exactly what the extension did until now.